src/app/downloads/downloads.component.html

```diff
@@ -14,9 +14,6 @@ <h2 class="card-title mb-3">Latest Release</h2>
           <h3 class="col-12 mb-2 mt-4 card-subtitle text-muted">Select your platform to start the download</h3>
           <div class="col-xs-12 col-sm-4 col-lg-4 mr-1">
             <a class="btn btn-default btn-primary btn-block btn-bg-dark mb-1" role="button" [href]="latestRelease.windows_main" (click)="trackDownload('windows', latestRelease.windows_main)">Windows (x64)</a>
-            <div class="alt-builds">
-              <a class="card-link text-center ml-2" [href]="latestRelease.windows_32" (click)="trackDownload('windows', latestRelease.windows_32)">Windows (x32)</a>
-            </div>
```

Comment:
While I could build a custom 32 bit installer, I am not having our CI do it anymore.  So I'm not advertising it.